SPECS-SIGNED/xpmem-modules-signed/xpmem-modules-signed.spec

```diff
@@ -16,7 +16,7 @@
 Summary:	 Cross-partition memory
 Name:		 xpmem-modules
```

Comment:
Should this also be suffixed with `-signed`?

Reply:
It could. In this PR I've focused mostly on making the SRPM names unique. We already have a few "signed" specs, which don't generate SRPMs with the "-signed" suffix, so I decided to minimize the changes needed to fix the duplicates issue, so that we maintain naming continuity in specs, which didn't need to be modified.

Let me know, if you'd like me to add the "-signed" suffix, though!

Reply:
I do think the two I've called out should have the "-signed" suffix in order to prevent these "signed" srpms from colliding with their unsigned counterparts